data/locale/attributes-pl.adoc

```diff
@@ -8,12 +8,12 @@
 :figure-caption: Rysunek
 :important-caption: Ważne
 :last-update-label: Ostatnio zmodyfikowany
-//ifdef::listing-caption[:listing-caption: ???]
+ifdef::listing-caption[:listing-caption: Spis]
```

Comment:
If by this we mean a block of code, then it is not `Spis`, but... to great surprise `Listing` :) `Spis` means more "a list", something written down, but not code print-out for whatever reason.

![eng](https://media.giphy.com/media/LqW9dLVjQm3cs/giphy.gif)